internal/handlers/not-found.go

```
@@ -0,0 +1,41 @@
+// 	thing := prepareThing()
+// 	return http.HandlerFunc(
+// 		func(w http.ResponseWriter, r *http.Request) {
+// 			// use thing to handle request
+// 			logger.Info(r.Context(), "msg", "handleSomething")
+// 		}
+// 	)
+// }
+
+func HandleNotFound(logger logger.Logger) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Wrong Path Bro\n"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)